src/test/skript/tests/syntaxes/expressions/ExprEntities.sk

```diff
@@ -1,11 +1,36 @@
 test "entities in chunk":
 	spawn 10 sheep at spawn of world "world"
-	wait 1 tick
-	assert size of all entities in chunk at spawn of world "world" >= 10 with "Size of all entities in spawn chunk is not > 10: %size of all entities in chunk at spawn of world "world"%"
+
+	set {_radius center} to location(-24, -60, -24, world "world")
+	spawn 10 sheep at {_radius center}
+
+	set {_corner::1} to location(20, -62, 20, world "world")
+	set {_corner::2} to location(28, -58, 28, world "world")
+	spawn 10 sheep at location(24, -60, 24, world "world")
```

Comment:
The tests happen instantaneously, so i'm not sure this test is actually getting any benefit from all the radius and within components of the syntax.

Potentially have fixed spawning location in a spread within the expected area to help this test be more precise.

Reply:
Im not sure how the instantaneous nature of tests affects the within andradius syntax. Could you elaborate on that? And what would fixed spawning look like because I thought I used fixed locations?

Reply:
Since it's instantaneous. The sheep won't spread like entity cramming pushing to spread them out, like you would think. They all just spawn on eachother and stay there, then they get removed, no tick is ever done to process their movement.

So all 10 sheep would have the same location. Having fixed spread locations would ensure that the radius checking is actually working and not just the center location where the sheep are.

Reply:
Ahh, that makes sense. I will add more locations